File: JavaScript. A3 Tasks/3 JavaScript для начинающих/3.02/3.2.03.js
/*


В этом задании в нашу функцию testStr первым параметром передается строка (переменная str), 
а вторым - число (переменная n) . Вам нужно вернуть из функции символ строки , порядковый номер которого указан в переданном в функцию числе.

Подсказка: порядковый номер не равен индексу символа в строке.

Sample Input:

Veryyeladygirlthemgoodmemake
24

Sample Output:

e

*/

function testStr(str, n) {
    return str[n-1];
}

let str = prompt()
let n = Number(prompt())
console.log(testStr(str, n))

File: Python. Part 2/step03/3.02/3.2.03.py
a = int(input())
b = int(input())
if b > a:
    for i in range(a, b+1):
        if 10 <= i <= 20:
            continue
        print(i)
else:
    for i in range(b, a+1):
        if 10 <= i <= 20:
            continue
        print(i)
